package.json

```
@@ -1,19 +1,33 @@
 {
-  "name": "screenshot_picker",
-  "version": "1.0.0",
-  "description": "web app to crawl file system and manage screenshots",
-  "main": "src/index.js",
-  "engines": {
-    "node": ">= 14",
-    "yarn": "^1.22.17"
-  },
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
-  "dependencies": {
-    "express": "^4.18.1"
-  }
+    "name": "screenshot_picker",
+    "version": "1.0.0",
+    "description": "web app to crawl file system and manage screenshots",
+    "main": "src/index.js",
+    "engines": {
+        "node": ">= 14",
+        "yarn": "^1.22.17"
+    },
+    "scripts": {
+        "test": "jest --watch",
+        "start": "cross-env NODE_ENV=development nodemon ./src/index.ts"
```

```suggestion
        "start": "nodemon ./src/index.js"
```
for now lets just stick with javascript... at a later point we can consider migrating to typescript.  I think for now we don't need to factor in a NODE_ENV yet, this will only be running locally not deployed to a prod environment.  also don't need cross-env I don't think